framespacer/R/asdataframe.R

```diff
@@ -0,0 +1,12 @@
+#' FrameSpace DataFrame to R DataFrame
+#' @description Converts a FrameSpace DataFrame into an R DataFrame
+#' @param dataframe \strong{list} FrameSpace DataFrame
+#' @return R DataFrame
+#' @export
+as.dataframe <- function(dataframe){
```

Comment:
I think you should change this to a method for `as.data.frame` that operates on a FrameSpace DataFrame class signature.